content/docs/csidriver/features/powerscale.md

```diff
@@ -527,3 +527,28 @@ In the case of operator-based installation, default permission for powerscale di
 
 Other ways of configuring powerscale volume permissions remain the same as helm-based installation.
 
+## Single Pod Access Mode for PersistentVolumes- ReadWriteOncePod (ALPHA FEATURE)
+
+Use `ReadWriteOncePod(RWOP)` access mode if you want to ensure that only one pod across whole cluster can read that PVC or write to it. This is only supported for CSI Driver for PowerScale 2.1 and Kubernetes version 1.22+.
```

Comment:
could write Dell EMC PowerScale 2.1.0 instead of PowerScale 2.1

Reply:
addressed

Reply:
This isnt addressed, probably u meant to comment else where.

Reply:
@randeepsharma https://github.com/dell/csm-docs/pull/90/commits/4faac67cf8ffad06224681cd744c3df73130acfa. Please refresh the link, it is addressed part of the above commit.